app/controllers/api/category_contents_controller.rb

```diff
@@ -0,0 +1,13 @@
+module API
+  class CategoryContentsController < APIController
+    def index
+      @primary_navigation, @secondary_navigation = Comfy::Cms::Category.navigation_categories
+      render json: @primary_navigation, scope: params[:locale].to_sym
```

Comment:
it's still on 2.1.2, anyway, the point is: try to whitelist user-provided values